### PR TITLE
Dark mode option color fix

### PR DESCRIPTION
**Pull request by muzafarova**
Fixes https://ubio-automation.atlassian.net/browse/AC-135

### CODE REVIEW

File: docs/index.css [hunk 1]
```diff
@@ -423,9 +423,9 @@ label[disabled] {
     opacity: .5;
 }
 select {
-    -moz-appearance: none;
     -webkit-appearance: none;
-    appearance: none;
+       -moz-appearance: none;
+            appearance: none;
```
Comment (kyungeunni) [inline docs/index.css]: Like this aligning 👍

File: stylesheets/base.css [hunk 2]
```diff
@@ -97,8 +97,6 @@ label[disabled] {
 }
 
 select {
-    -moz-appearance: none;
```
Comment (kyungeunni) [inline stylesheets/base.css]: Is it safe to delete them?? is it because it's duplicated in `index.css` ?

Comment (muzafarova) [reply stylesheets/base.css]: yes, safe! It's a source file that later gets through compiling process and the final outcome that we deliver (/index.css) is having all the prefixes 🎉

File: docs/index.css (hunk 1 above)
Comment (muzafarova) [reply docs/index.css]: it's automatic :D

**Pull request merged**